quinn-proto/src/connection/mod.rs

```diff
@@ -2144,6 +2143,19 @@ where
             };
 
             self.stats.frame_rx.record(&frame);
+            // Crypto and Stream frames are special cased in order no pollute
+            // the log with payload data
+            match &frame {
+                Frame::Crypto(f) => {
+                    trace!(offset = f.offset, len = f.data.len(), "got crypto frame");
+                }
+                Frame::Stream(f) => {
+                    trace!(id = %f.id, offset = f.offset, len = f.data.len(), fin = f.fin, "got stream frame");
+                }
+                f => {
+                    trace!("got frame {:?}", f);
```

Comment:
```suggestion
                    trace!("got frame");
```
The frame type is already recorded in the span.

Reply:
Or are you intentionally including the frame payload?

Reply:
Yes. For all the `MAX_` frames it is interesting to know what the limits are. The payloads should never be too long apart for CRYPTO and STREAM, which are special cased for this reason

Reply:
Ok, guess we should special case `DATAGRAM` too